lib/python/qmk/commands.py

```diff
@@ -61,3 +64,22 @@ def parse_configurator_json(configurator_file):
     user_keymap = json.load(configurator_file)
 
     return user_keymap
+
+
+def run(command, **kwargs):
```

Comment:
Best practice when wrapping another function is to pass both `*args` and `**kwargs`.

```suggestion
def run(command, *args, **kwargs):
```